rust/salvo/Cargo.toml

```diff
@@ -6,4 +6,4 @@ authors = ["Chrislearn Young <chrislearn@hotmail.com>"]
 
 [dependencies]
 salvo = { version = "~0.5" }
-tokio = { version = "1.0", features = ["full"] }
+tokio = { version = "1.1", features = ["full"] }
```

Comment:
any developer using `salvo` have to ping its `tokio` version. should not in be handled by `salvo` like in https://github.com/gotham-rs/gotham/blob/master/gotham/Cargo.toml#L32 @chrislearn ?